Makefile

```diff
@@ -55,6 +55,12 @@ e2e-test:
 
 	./tests/run-all.sh
 
+# Run e2e tests against the configured Kubernetes cluster in ~/.kube/config
```

Comment:
Local cluster or do we start using Kind?

Reply:
Any cluster that you are logged in (this info is stored in `~/.kube/config`)